.changeset/modern-hornets-jam.md

```diff
@@ -0,0 +1,32 @@
+---
+"@latticexyz/cli": major
+"@latticexyz/store": major
+"@latticexyz/world": major
+"@latticexyz/store-sync": major
+"create-mud": patch
+---
+
+- BREAKING: `Store`'s internal Schema table is now a normal table instead of using special code paths. It is renamed to Tables, and the table id chagned from `mudstore:schema` to `:TableMetaata`
```

Comment:
```suggestion
- BREAKING: `Store`'s internal schema table is now a normal table instead of using special code paths. It is renamed to Tables, and the table ID changed from `mudstore:schema` to `mudstore:Tables`
```

Reply:
I haven't been prefixing anything with "BREAKING", assuming that the `major` tag does this. Should we be marking changesets like this?

Reply:
I saw this in the viem changelog but actually agree that major changes are by definition breaking so this is redundant. (Especially if we change the headlines from patch/minor/major to fix/feature/breaking or something like that)